internal/pkg/describe/webapp.go

```diff
@@ -68,7 +76,27 @@ func NewWebAppDescriber(project, app string) (*WebAppDescriber, error) {
 	}, nil
 }
 
-// URI returns the stringified WebAppURI to identify this application uniquely given an environment name.
+// DeployInfo returns the deploy infomation of a web application given an environment name.
+func (d *WebAppDescriber) DeployInfo(envName string) (*WebAppDeployInfo, error) {
```

Comment:
following the comment above: `func (d *WebAppDescriber) ECSParams(envName string) (*WebAppECSParams, error)`